core/vm/evm.go

```diff
@@ -526,6 +525,33 @@ func (evm *EVM) Create2(caller ContractRef, code []byte, gas uint64, endowment *
 // ChainConfig returns the environment's chain configuration
 func (evm *EVM) ChainConfig() *params.ChainConfig { return evm.chainConfig }
 
+func getTobinTax(evm *EVM, sender common.Address) (*big.Int, *big.Int, *common.Address, error) {
```

Comment:
Is this file from upstream geth? If so, do you think we could take advantage of this being an independent function, and move it to a different file, to make future merges with upstream easier (lessening the amount of differences to check)

Reply:
Are we likely to see merge conflicts here? All functions from here on down are additions with respect to upstream, I'm not sure if breaking this out into a separate file will move the needle